src/Utils/getCroppedImg.tsx

```diff
@@ -0,0 +1,35 @@
+const createImage = (url: string): Promise<HTMLImageElement> =>
+  new Promise((resolve, reject) => {
+    const image = new Image();
+    image.addEventListener("load", () => resolve(image));
+    image.addEventListener("error", (error) => reject(error));
+    image.setAttribute("crossOrigin", "anonymous"); // needed to avoid cross-origin issues on some browsers
+    image.src = url;
+  });
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function getCroppedImg(imageSrc: string, croppedAreaPixels: any) {
+  const image = await createImage(imageSrc);
+  const canvas = document.createElement("canvas");
+  const ctx = canvas.getContext("2d");
+
+  if (!ctx) {
+    return null;
```

Comment:
```suggestion
    return Promise.resolve("");
```

Maintain consistent return type.